src/wallet/coinselection.h

```diff
@@ -93,6 +95,8 @@ struct OutputGroup
     void Insert(const CInputCoin& output, int depth, bool from_me, size_t ancestors, size_t descendants);
     std::vector<CInputCoin>::iterator Discard(const CInputCoin& output);
     bool EligibleForSpending(const CoinEligibilityFilter& eligibility_filter) const;
+    void SetFees(const CFeeRate effective_feerate, const CFeeRate long_term_feerate);
+    OutputGroup GetPositiveOnlyGroup();
```

Comment:
My understanding from #17526 was that one instance of `OutputGroup` described a single UTXO with its ancestry, which seems consistent with the struct above.
It seems to me though, that the function `GetPositiveOnlyGroup()` operates on a set of OutputGroup in the above sense, though. Is that the case?

Reply:
No, `OutputGroup` is a group of UTXOs that we want to consider together. It doesn't include ancestry. IIRC this is used for avoid reuse and for some privacy stuff.

Reply:
I see, thanks. I wasn't thinking about the bundling of all utxos associated with one address.